### PR TITLE
Preloaded images

### PR DESCRIPTION
**Pull request by yozhio**
We load images and convert URLs when loading html pages.

### CODE REVIEW

File: scripts/lib/html_normalizer.py [hunk 1]
```diff
@@ -123,11 +131,64 @@ def flag_potential_footnotes(
...
+            continue
+
+        for size_re in IMG_RE["size_attr"], IMG_RE["size_style"]:
+            sizes = dict(size_re.findall(attrs))
+
+            if not len(sizes):
+                continue
+
+            for k in sizes:
+                if sizes[k][-1].isnumeric():
```
Comment (worldmind) [inline scripts/lib/html_normalizer.py:146]: What if not numeric?

**Review by worldmind — APPROVED** [review]
Looks good

File: scripts/lib/html_normalizer.py (hunk 1 above)
Comment (yozhio) [reply scripts/lib/html_normalizer.py:146]: if it is not numeric, we already have a unit in the string

Comment (worldmind) [reply scripts/lib/html_normalizer.py:146]: Perhaps good to add that as a comment

**Pull request merged**